float motor_state
---
bool state
